paddle/fluid/operators/ngraph/ngraph_ops.h

```diff
@@ -22,6 +22,7 @@ limitations under the License. */
 #pragma once
 
 #include "ops/binary_unnary_op.h"
+#include "ops/elementwise_add_op.h"
```

Comment:
How about auto-generate this file like
https://github.com/PaddlePaddle/Paddle/blob/6ca9a4810b53f9aa750e9c2536ef6f70010e5618/paddle/fluid/framework/ir/CMakeLists.txt#L1-L4